api/src/routes/sanitize.ts

```diff
@@ -15,4 +15,4 @@ export function sanitizeParameters(customRules = {}, defaultRule = /^[-.a-zA-Z0-
     };
 }
 
-export const pidSanitizer = sanitizeParameters({ pid: /^[a-zA-Z]+:[0-9]+/ }, /^$/);
+export const pidSanitizer = sanitizeParameters({ pid: /^[a-zA-Z]+:[0-9]+/ });
```

Comment:
I wonder if we need to rethink the sanitizeParameters function to make it easier to chain things. For example, maybe if we pass null or false to defaultRule, it entirely bypasses checking for parameters outside of customRules. That would allow us to use the defined pidSanitizer to sanitize PIDs, but then chain a new purpose-built function created by sanitizeParameters to handle any additional parameters.

There are really a lot of different ways I could think of to approach this, and I'm not sure what's best... but I want to be sure we don't paint ourselves into a corner.

Reply:
I removed the regex "^$" because it was disallowing all values from additional params. There many ways to implement this. Atm, I can't think up those options. I think it requires further investigation and maybe should be delegated to another task.

Reply:
How do you feel about b7417b3398603a4a4f4593e34fce64aa1bc192a9? This gets the behavior we need without changing as much existing behavior. There's still probably room to refactor/simplify, but I think this makes the current PR a little bit less impactful, and we can optimize the rest later.

Reply:
Sure. Let's also create a followup story.

Reply:
I've created a task for us in Teams. We may at some point want to begin moving some things from Teams to GitHub issues if it makes more sense to discuss them in a more public setting. At this early stage, though, I don't think it makes a difference yet.